src/data/finset/powerset.lean

```diff
@@ -160,7 +160,9 @@ lemma powerset_len_sup [decidable_eq α] (u : finset α) (n : ℕ) (hn : n < u.c
   (powerset_len n.succ u).sup id = u :=
 begin
   apply le_antisymm,
-  { simp [mem_powerset_len, and.comm] },
+  { simp_rw [sup_le_iff, mem_powerset_len],
+    rintros x ⟨h, -⟩,
+    exact h },
```

Comment:
This is a little worrying. I think `sup_le_iff` no longer fires now that `le_iff_subset` was replaced with `le_eq_subset`, but I'm not sure what the best solution is.